examples/awq/qwen3_moe_example.py

```diff
@@ -3,13 +3,12 @@
 
```

Comment:
Instead of deleting the entire big_models_with_accleerate folder, I would keep a readme highlighting/summarizing how the models are handled now  i.e cover the basics of your PR description